src/faultlog/faultlog_main.cpp

```diff
@@ -200,6 +200,17 @@ void propertyChanged(sdbusplus::bus::bus& bus, sdbusplus::message::message& msg,
     }
 }
 
+void addServiceableEvents(const nlohmann::json& errorlog,
```

Comment:
/** @brief 
 *
 *  @param[in] 
 *  @param[in] 
  */

Please add the function header